src/main/java/com/iexec/worker/tee/post/compute/signer/SignerService.java

```diff
@@ -20,8 +20,10 @@ public static String signEnclaveChallenge(String messageHash, String enclaveChal
                 CredentialsUtils.getAddress(enclaveChallengePrivateKey));
 
         if (!isSignatureValid) {
-            log.error("Failed to verify TeeEnclaveChallenge signature (exiting)");
-            throw new PostComputeException(POST_COMPUTE_INVALID_TEE_SIGNATURE);
+            throw new PostComputeException(
+                    POST_COMPUTE_INVALID_TEE_SIGNATURE,
+                    "Failed to verify TeeEnclaveChallenge signature (exiting)"
```

Comment:
Note: Message is particularly useful if a same statusCause is thrown in multiple locations or if some dynamic parameters are added to the message.